ferveo-python/ferveo/__init__.pyi

```diff
@@ -173,26 +177,24 @@ class SharedSecret:
         ...
 
 
+@final
 class FerveoVariant:
-    @staticmethod
-    def simple() -> str: ...
-
-    @staticmethod
-    def precomputed() -> str: ...
+    simple: str
+    precomputed: str
 
 
-def encrypt(message: bytes, add: bytes, dkg_public_key: DkgPublicKey) -> Ciphertext:
+def encrypt(message: bytes, aad: bytes, dkg_public_key: DkgPublicKey) -> Ciphertext:
     ...
 
 
 def combine_decryption_shares_simple(
-        decryption_shares: Sequence[DecryptionShareSimple],
+        shares: Sequence[DecryptionShareSimple],
```

Comment:
Does changing this parameter name lead to more downstream changes to `ferveo` and `nucypher`?

I guess I'm wondering, how strongly do you feel about changing the parameter from `decryption_shares` to `shares` if it has downstream effects...?

Reply:
I didn't consider it having a downstream effect, i.e. didn't consider the usage of named parameters on `nucypher`. I think I can simply revert this change.